api/api.go

```diff
@@ -26,14 +26,21 @@ var (
 	// metric api.iters_to_points is how long it takes to decode points from a chunk iterator
 	itersToPointsDuration = stats.NewLatencyHistogram15s32("api.iters_to_points")
 
-	// metric api.request_handle is how long it takes to handle a render request
-	reqHandleDuration = stats.NewLatencyHistogram15s32("api.request_handle")
-
 	// metric api.requests_span.mem_and_cassandra is the timerange of requests hitting both in-memory and cassandra
 	reqSpanBoth = stats.NewMeter32("api.requests_span.mem_and_cassandra", false)
 
 	// metric api.requests_span.mem is the timerange of requests hitting only the ringbuffer
 	reqSpanMem = stats.NewMeter32("api.requests_span.mem", false)
+
+	// metric api.render.series is the number of series a /render request is handling.  This is the number
+	// of metrics after all of the targets in the request have expaned by searching the index.
+	reqRenderSeriesCount = stats.NewMeter32("api.render.series", false)
+
+	// metric api.render.series is the number of targets a /render request is handling.
+	reqRenderTargetCount = stats.NewMeter32("api.render.targets", false)
+
+	// metric api.render.series is the number of points that need to be fetched for a /render request.
+	reqRenderPointsCount = stats.NewMeter32("api.render.points", false)
```

Comment:
good point.  I didnt notice we collected this metric as it is not on the dashboards or docs.  It seems to be good indicator of the query workload so we should put on the dashboard IMHO.

So we can keep emitting that metric from the alignRequests function, but i have renamed it to be more consistent with the other render query metrics.

So I have kept the existing